DockerfileLinuxWheelCreation

```diff
@@ -0,0 +1,23 @@
+# Stage 1: Preparing image having SEAL
+FROM alpine:3.10.2 AS buildseal
+RUN apk add --no-cache git g++ make cmake patchelf libstdc++ libgomp python3 py3-cffi py3-numpy python3-dev
+# Build SEAL
+RUN mkdir /building && \
+    cd /building && \
+    git clone --single-branch --branch 3.3.1 https://github.com/microsoft/SEAL.git && cd SEAL/native/src && \
+    cmake . && make install -j8
+
+# Stage 2: Build python wheel
+FROM buildseal AS buildwheel
```

Comment:
There isn't any reason to have a stage 2 that just directly keeps building on stage 1? Shouldn't this be a new base and then `COPY` across the built SEAL library

Reply:
In stage 1, we are not only building the SEAL library, but also installing it, and al the required python dependencies.
In stage 2, the library is required for building the library and the wheel.
In stage 3, SEAL is still required to run the installed wheel but we do not need the python code, only the wheel created from stage 2.
So we use the same base created from stage 1 (installing SEAL) for the two following stages.

Reply:
But it's true we could delete all the SEAL installation process as done in the original dockerfile, but as the goal here is to create a working wheel, I didn't really thin about cleaning too much.

Reply:
Ahh okay I follow now